packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -350,17 +369,53 @@ export class BasePicker<T extends {}, P extends IBasePickerProps<T>>
             )}
           </div>
         </SelectionZone>
+        {this.renderError(classNames.error)}
         {this.renderSuggestions()}
       </div>
     );
   }
 
+  protected _getDescribedBy = (items: T[], hasError: boolean): string => {
+    const describedBy = [];
+    if (items.length > 0) {
+      describedBy.push(this._ariaMap.selectedItems);
+    }
+    if (hasError) {
+      describedBy.push(this._ariaMap.error);
+    }
+    return describedBy.join(' ');
+  };
```

Comment:
_nit:_ You could also achieve this, possibly even with better performance, by doing simple string concatenation instead of pushing into an array and joining.